manifests/dm/master/dm-master.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/component: dm-master
+  name: dm-master
+spec:
+  podManagementPolicy: Parallel
+  replicas: 3
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: dm-master
+  serviceName: dm-master-peer
+  template:
+    metadata:
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "8261"
+        prometheus.io/scrape: "true"
+      creationTimestamp: null
+      labels:
+        app.kubernetes.io/component: dm-master
+    spec:
+      affinity: {}
+      containers:
+      - command:
+        - /dm-master
+        - -data-dir=/data
+        - --name=$(MY_POD_NAME)
+        - --master-addr=:8261
+        - --advertise-addr=$(MY_POD_NAME).$(PEER_SERVICE_NAME).$(NAMESPACE):8261
+        - --peer-urls=:8291
+        - --advertise-peer-urls=http://$(MY_POD_NAME).$(PEER_SERVICE_NAME).$(NAMESPACE):8291
+        - --initial-cluster=dm-master-0=http://dm-master-0.$(PEER_SERVICE_NAME).$(NAMESPACE):8291,dm-master-1=http://dm-master-1.$(PEER_SERVICE_NAME).$(NAMESPACE):8291,dm-master-2=http://dm-master-2.$(PEER_SERVICE_NAME).$(NAMESPACE):8291
+        env:
+        - name: MY_POD_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.name
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+        - name: PEER_SERVICE_NAME
+          value: dm-master-peer
+        - name: SERVICE_NAME
+          value: dm-master
+        - name: TZ
+          value: UTC
+        image: 127.0.0.1:30002/pingcap/dm:2040
```

Comment:
There is already a DM ha-alpha version on DockerHub.

Reply:
Updated.